kernel/src/expressions/mod.rs

```diff
@@ -50,25 +50,8 @@ impl BinaryOperator {
             GreaterThanOrEqual => Some(LessThanOrEqual),
             LessThan => Some(GreaterThan),
             LessThanOrEqual => Some(GreaterThanOrEqual),
-            Equal | NotEqual | Plus | Multiply => Some(*self),
-            _ => None,
-        }
-    }
-
-    /// invert an operator. Returns Some<InvertedOp> if the operator supports inversion, None if it
-    /// cannot be inverted
-    pub(crate) fn invert(&self) -> Option<BinaryOperator> {
```

Comment:
nit: Should this just consume `self`?

Reply:
Originally it took `&self` because the caller never had an owned copy laying around. But now the code is completely deleted.